### PR TITLE
all: use gonum.org/v1/plot/cmpimg

### PR DESCRIPTION
**Pull request by sbinet**

### CODE REVIEW

**Comment by jaybatra26** [comment]
@sbinet, What advanctage was `"gonum.org/v1/plot/cmpimg` providing that `go-hep.org/x/hep/hplot/cmpimg` was not providing. Just a curious question :)
	"

File: groot/rsrv/rsrv_test.go [hunk 1]
```diff
@@ -25,17 +24,13 @@ import (
 	"time"
 
 	uuid "github.com/hashicorp/go-uuid"
-	"go-hep.org/x/hep/hplot/cmpimg"
+	"gonum.org/v1/plot/cmpimg"
```
Comment (jaybatra26) [inline groot/rsrv/rsrv_test.go]: @sbinet, What advanctage was `"gonum.org/v1/plot/cmpimg` providing that `go-hep.org/x/hep/hplot/cmpimg` was not providing? Just a curious question :)

Comment (jaybatra26) [inline groot/rsrv/rsrv_test.go]: @sbinet, What advanctage was `"gonum.org/v1/plot/cmpimg` providing that `go-hep.org/x/hep/hplot/cmpimg` was not providing? Just a curious question :)

**Comment by sbinet** [comment]
`hplot/cmpimg` was an old copy of `gonum/plot/internal/cmpimg`, _i.e._ _sans_ a couple of bug fixes.
now that `gonum/plot/cmpimg` is exported, might as well use it directly instead of "vendoring" it :)

**Comment by jaybatra26** [comment]
@sbinet, Thanks :)

**Pull request merged**